Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1526,12 +1526,22 @@ - (void)removeAllVideoTargets;
 {
     ASSERT(m_synchronizer);
     m_isGatheringVideoFrameMetadata = true;
+    if (RefPtr videoRenderer = m_videoRenderer) {
+        videoRenderer->notifyWhenHasAvailableVideoFrame([weakThis = WeakPtr { *this }](const MediaTime& presentationTime, double displayTime) {
+            if (RefPtr protectedThis = weakThis.get()) {
+                protectedThis->setHasAvailableVideoFrame(true);
```

Comment:
Why do we need this line? Isn't it guaranteed by `notifyFirstFrameAvailable` call below?